composer.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "pressbooks/pressbooks",
+  "license" : "GPL-2.0+",
+  "type" : "wordpress-plugin",
+  "description": "Pressbooks is an open source book publishing tool built on a WordPress multisite platform. Pressbooks outputs books in multiple formats, including PDF, EPUB, MOBI, web, and a variety of XML flavours, using a theming/templating system, driven by CSS.",
+  "homepage": "https://github.com/pressbooks/pressbooks",
+  "authors": [
+    {
+      "name": "Book Oven Inc.",
+      "email": "code@pressbooks.com",
+      "homepage": "https://pressbooks.com"
+    }
+  ],
+  "keywords": [
+    "ebooks publishing webbooks"
+  ],
+  "support": {
+    "email": "code@pressbooks.com",
+    "issues": "https://github.com/pressbooks/pressbooks/issues",
+    "docs": "https://github.com/pressbooks/pressbooks/wiki"
+  },
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/pressbooks/htmlawed"
+    },
+    {
+      "type": "vcs",
+      "url": "https://github.com/pressbooks/PrinceXMLPhp"
+    }
+  ],
```

Comment:
[Good use of forking](http://snippets.khromov.se/composer-use-your-own-fork-for-a-package/). .